flow_screen_components/LightningInputFSC/README.md

```diff
@@ -0,0 +1,126 @@
+# Lightning Input
+This is a port of the [Lightning Input](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/aura_compref_lightning_input.htm) component, which includes the following input types:
+
+checkbox | checkbox-button | color | date | datetime-local | email | month | number | password | radio | range | tel | time | toggle | url | week
+
+## How It Works ##
+
+The component has two required attributs: `type` (possible values noted above) and `label`, which must be set in the Flow. There are also several type-specific attributes and validation attributes available ([see here for full documentation]((https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/aura_compref_lightning_input.htm)))
+
+A `value` attribute captures the user's input and can be used to set a Flow variable.
+
+For checkboxes and radio buttons, setting the "Group Name" attribute in the Flow will tie the buttons together.
+
+## Attributes and Examples by Type
+
+* type: "checkbox"
+   * label: "Red" 
+   * checked: "true"
+
+* type: "checkbox"
+   * label: "Blue" 
+
+* type: "checkbox"
+   * label: "Add pepperoni" 
+   * checked: "true" 
+   * value: "pepperoni" 
+
+* type: "checkbox-button" 
+  * label: "Add salami" 
+  * value: "salami" 
+
+* type: "color"
+   * label: "Color" 
+   * value: "#EEEEEE"
+
+* type: "date"
+   * label: "Birthday" 
+
+* type: "datetime-local" 
+  * label: "Birthday" 
+
+* type: "email"
+   * label: "Email" 
+   * value: "abc@domain.com" 
+
+* type: "month"
+   * label: "Birthday" 
+
+* type: "number"
+   * label: "Number" 
+   * value: "12345"
+
+* type: "number"
+   * label: "Enter a number" 
+
+* type: "number"
+   * label: "Enter a decimal value" 
+   * step: "0.001"
+
+* type: "number"
+   * label: "Enter a percentage value" 
+   * formatter: "percent" 
+   * step: "0.01" 
+
+* type: "number"
+   * label: "Enter a dollar amount" 
+   * formatter: "currency" 
+   * step: "0.01" 
+
+* type: "password"
+   * label: "Password" 
+
+* type: "radio"
+   * label: "Red" 
+   * value: "red" 
+   * checked: "true" 
+
+* type: "radio"
+   * label: "Blue" 
+   * value: "blue" 
+
+* Radio Group
+  * type: "radio"
+    * label: "Red" 
+    * group name: "color" 
+    * value: "red" 
+    * checked: "true" 
+
+  * type: "radio"
+    * label: "Blue" 
+    * group name: "color" 
+    * value: "blue" 
+
+
+* type: "range"
+   * label: "Number" 
+   * min: "0" 
+   * max: "10"
+
+* type: "tel"
+   * label: "Telephone" 
+   * value: "343-343-3434" 
+   * pattern: "[0-9]{3}-[0-9]{3}-[0-9]{4}"
+
+* type: "time"
+   * label: "Time" 
+
+* type: "toggle"
+   * label: "Toggle value" 
+   * checked: "true" 
+
+* type: "url"
+   * label: "Website" 
+
+* type: "week"
+   * label: "Week" 
+
+## Custom Validation Parameters and Messages
+
+### The following validations can be set in the Flow to show default or custom messages:
+formatter | max | maxlength | min | minlength | required 
```

Comment:
You've added a lot of richness here, and some of it needs a little more doc. For example, for "formatter", it would be great to show the three legit values and give an example.